Automattic/wp-calypso
# Adjustments to plans step in `onboarding-with-email` signup flow

#### Proposed Changes

Letero is working on a set of improvements to the onboarding-with-email signup flow. This flow captures users that have shown interest in the email solutions offered on WordPress.com. For more context, see pbLl1t-1f6-p2.

In this PR, I have made a few changes to the plans step in that signup flow:

1. Added a variation of the plans step: `mailbox-plan`
2. Changed the title in that step to `Want a website to expand your online presence?`
3. Include a "skip" button in the subtitle that reads `continue with setting up your Professional Email`. This will allow users to proceed with the Free plan.
4. Customized the thank you page after users complete checkout

#### Testing Instructions



**1️⃣ Test the onboarding-with-email flow**

1. Navigate to `/start/onboarding-with-email`
2. Search for a domain, proceed to next step
3. Click "Add" in the emails step
4. Ensure that the heading in the next step reads `Want a website to expand your online presence?`
5. Ensure that there is a `continue with setting up your Professional Email` button in the subheader
6. Click that button. Ensure that you are taken to checkout and that you have only a domain and a Professional Email subscription in your cart.
7. Complete checkout
8. Ensure that you are taken to `/checkout/thank-you/features/email-license/:siteSlug/:receiptId?siteId=:siteId`
9. Ensure that the page looks like the screenshot below

![thank you](https://user-images.githubusercontent.com/1101677/188140713-bab86bb3-d658-41a6-b684-646cd4a3a574.png)

**2️⃣ Test the regular flow for regressions**
1. Navigate to `/start`
2. Proceed to the plans step
3. Ensure that the heading reads `Choose a plan`
4. Ensure that there is no `start with a free site` button in the subheader

#### Pre-merge Checklist






Related to #66921 #67046 #67325
